client/blocks/inline-help/contextual-help.js

```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
-			title: 'Integrate and Connect to Social Media',
-			description:
+			title: translate( 'Integrate and Connect to Social Media' ),
+			description: translate(
 				'Start sharing your site and attract more traffic and visitors to your content! ' +
-				'Learn to activate and control the social media and sharing options on your website or blog through these videos.',
+					'Learn to activate and control the social media and sharing options on your website or blog through these videos.'
+			),
 		},
 		{
 			link: 'https://en.support.wordpress.com/sharing/',
 			post_id: 7499,
-			title: 'Sharing Your Content',
-			description:
+			title: translate( 'Sharing Your Content' ),
+			description: translate(
 				'At the bottom of each post or page, you can include sharing buttons for your readers ' +
-				'to make it easier to share your content.',
+					'to make it easier to share your content.'
+			),
 		},
 		{
 			link: 'https://en.support.wordpress.com/instagram/',
 			post_id: 77589,
-			title: 'Using Instagram',
-			description:
+			title: translate( 'Using Instagram' ),
+			description: translate(
 				'Instagram is a simple way to capture, customize, ' +
-				'and share photos and short videos using your smartphone or other mobile device. Learn how to use it with your website!',
+					'and share photos and short videos using your smartphone or other mobile device. Learn how to use it with your website!'
+			),
 		},
 		{
 			link: 'https://en.support.wordpress.com/twitter/',
 			post_id: 124,
-			title: 'Using Twitter',
-			description:
+			title: translate( 'Using Twitter' ),
+			description: translate(
 				'Twitter is a service for the exchange of brief messages, commonly ' +
-				'called "tweets", between users. Learn how to use it with your website!',
+					'called "tweets", between users. Learn how to use it with your website!'
+			),
 		},
 	],
 	me: [
 		{
 			link: 'http://en.support.wordpress.com/manage-my-profile/',
 			post_id: 19775,
-			title: 'Managing Your Profile',
-			description:
+			title: translate( 'Managing Your Profile' ),
+			description: translate(
 				'Your profile is the information you’d like to be shown along with your ' +
-				'name when you publish content or comment on WordPress.com sites.',
+					'name when you publish content or comment on WordPress.com sites.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/gravatars/',
 			post_id: 1338,
-			title: 'Your Profile Picture',
-			description:
+			title: translate( 'Your Profile Picture' ),
+			description: translate(
 				'WordPress.com associates an Avatar with your email address. Gravatar ' +
-				'powers the user avatars on WordPress.com.',
+					'powers the user avatars on WordPress.com.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/account-deactivation/',
 			post_id: 138080,
-			title: 'Account Closure',
-			description: 'Need a fresh start? Learn how to close your account.',
+			title: translate( 'Account Closure' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Confirm account closure' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).